pkg/webhook/cm/plugins/sloconfig/slo_config_plugin.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sloconfig
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+	ctrladmission "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/koordinator-sh/koordinator/pkg/util/parallelize"
+	"github.com/koordinator-sh/koordinator/pkg/util/sloconfig"
+)
+
+const (
+	PluginName = "SLOControllerConfig"
```

Comment:
It is better to name it `SLOConfig`, consistent with the package name

Reply:
done